### DISCUSSION TITLE
Adjust the brightness of the external display black screen via VGA connection

### DISCUSSION THREAD

**Discussion by PromiseFor** [MonitorControl/MonitorControl#1168] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Question

When I open the app to adjust the brightness, the additional monitor goes black and then lights up again, and the brightness of the additional monitor does not change;
Is the VGA interface not supported?
Or the monitor is too old?

**Comment by waydabber** [comment]
VGA is generally supported with the right dongle but the display itself might react badly to DDC commands (not all VGA monitors support DDC as this is the case with HDMI and DisplayPort monitors as well).
